aten/src/ATen/function_wrapper.py

```diff
@@ -190,6 +213,19 @@ def TypedDict(name, attrs, total=True):  # type: ignore
 }
 """)
 
+C10_FUNCTION_DEFINITION = CodeTemplate("""\
+static inline ${return_type} ${api_name}(${formals}) {
+#ifdef USE_STATIC_DISPATCH
+    ${static_dispatch_function_body}
+#else
+    static c10::OperatorHandle op = c10::Dispatcher::singleton()
+        .findSchema({"aten::${name}", "${overload_name}"}).value();
+    return c10::Dispatcher::singleton().callUnboxed<${formals_types_with_return}>(
+        op, impl::dispatchTypeId(${inferred_type_set})${native_actuals_with_comma_prefix});
```

Comment:
Also going to need multiple dispatch bits too.

Reply:
this is using `inferred_type_Set`, same as the aten version, so should be ok, right?

Ultimately we want to move this logic into the dispatcher, but I'm waiting for more of your dispatch diffs to land before doing that because it would likely conflict.

Reply:
Multidispatch is landed, so all of the stuff you need is landed already.